### PR TITLE
initial tests for monolingual property T88546

### PR DESCRIPTION
**Pull request by WMDE-Fisch**
see https://phabricator.wikimedia.org/T88546

### CODE REVIEW

File: tests/browser/features/step_definitions/statement_steps.rb [hunk 1]
```diff
@@ -62,6 +62,16 @@
   end
 end
 
+When /^I enter (.+) in the InputExtender input field$/ do |value|
+  on(ItemPage).inputextender_input_element.when_visible.value = "#{value}"
```
Comment (zeljkofilipin) [inline tests/browser/features/step_definitions/statement_steps.rb]: There is no need for string interpolation here. `"#{value}"` can be just `value`

**Comment by zeljkofilipin** [comment]
:+1: in general, one minor comment.

File: tests/browser/features/step_definitions/statement_steps.rb [hunk 2]
```diff
@@ -258,3 +272,7 @@
 Then(/^(.*) should be the geo precision setting$/) do |precision|
   expect(on(ItemPage).geo_precision_element.when_present.text).to eq precision
 end
+
+Then /^InputExtender dropdown should be there$/ do
```
Comment (thiemowmde) [inline tests/browser/features/step_definitions/statement_steps.rb]: Why do some of the `Then` lines do use brackets and others do not?

Comment (WMDE-Fisch) [inline tests/browser/features/step_definitions/statement_steps.rb]: Very good question. Never really noticed. Actually there is a convention also checked by rubocop saying that regexp literals should be paranthesized. We deactivated that check. 

I will open a new pull request adding brackets everywhere and activating the test in rubocop.

File: tests/browser/features/statements_monolingual.feature [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      And I enter something in the claim value input field
+      And I enter English in the InputExtender input field
+      And I click the InputExtender dropdown first element
+      And I click the statement save button
+      And I reload the page
+    Then Statement string value of claim 1 in group 1 should be something (English)
+      And Statement name of group 1 should be the label of monolingprop
+      And Statement save button should not be there
+      And Statement cancel button should not be there
+      And Statement edit button for claim 1 in group 1 should be there
```
Comment (thiemowmde) [inline tests/browser/features/statements_monolingual.feature]: Is there a convention and a automated check for having a newline at the end? Would be nice.

Comment (zeljkofilipin) [inline tests/browser/features/statements_monolingual.feature]: RuboCop should be checking for that. Is it set up for this repo?

Comment (WMDE-Fisch) [inline tests/browser/features/statements_monolingual.feature]: Yes RuboCop is set up and normally it should check that by default. I experimented with adding the corresponding parameters to the .rubocop,yml. But RuboCop sill passes even with the violation. ... strange. Could a bug in RuboCop...

**Pull request merged**